src/test/java/suite/optout/OptoutTest.java

```diff
@@ -151,10 +170,56 @@ public void testV2TokenRefreshAfterOptOut(String label, Operator operator, Strin
         with().pollInterval(5, TimeUnit.SECONDS).await("Get V2 Token Response").atMost(OPTOUT_WAIT_SECONDS, TimeUnit.SECONDS).until(() -> operator.v2TokenRefresh(refreshToken, refreshResponseKey).equals(Mapper.OBJECT_MAPPER.readTree("{\"status\":\"optout\"}")));
     }
 
+    @Order(9)
+    @ParameterizedTest(name = "/v2/optout/status after v2/identity/map and v2/token/logout - DII {0} - expecting {4} - {2}")
+    @MethodSource({"afterOptoutAdvertisingIdArgs"})
```

Comment:
To clarify: this is getting updated by running previous tests first - specifically `testV2LogoutWithV2IdentityMap()`

Can't the assertions here be applied to existing tests?

Reply:
are you suggesting I merge the two tests ? Yes I can merge them. Having separate tests has the advantage that the test needs to wait for less time for the opt out data to flow through, since there are other methods which run before it.
Do you see any benefit in merging them ?

Reply:
Nope that makes sense, thanks for the clarification